src/sdk/pynni/nni/nas/pytorch/enas/mutator.py

```diff
@@ -30,11 +30,12 @@ def forward(self, inputs, hidden):
 class EnasMutator(Mutator):
 
     def __init__(self, model, lstm_size=64, lstm_num_layers=1, tanh_constant=1.5, cell_exit_extra_step=False,
-                 skip_target=0.4, branch_bias=0.25):
+                 skip_target=0.4, temperature=None, branch_bias=0.25, entropy_reduction="sum"):
```

Comment:
do you need to change the doc here? since the user will implement a customized EnasMutator

Reply:
Let's do it separately.